beetsplug/lyrics.py

```diff
@@ -16,17 +16,18 @@
 """Fetches, embeds, and displays lyrics.
 """
 
-from __future__ import division, absolute_import, print_function
+from __future__ import absolute_import, division, print_function
 
+import difflib
+import itertools
+import json
 import re
 import requests
-import json
 import unicodedata
 import urllib
-import difflib
-import itertools
 import warnings
 from HTMLParser import HTMLParseError
+from langdetect import detect
```

Comment:
This should be conditionally imported, so the library isn't required if you don't enable translation.